packages/core/src/rum/instrumentation/resourceTracking/distributedTracing/distributedTracing.tsx

```diff
@@ -77,55 +109,177 @@ const generateTracingAttributesWithSampling = (
+     * @param length - The desired length of the output string.
+     * @returns The ID as a padded string in the specified radix.
+     */
+    public toPaddedString(radix: number, length: number): string {
+        return BigInt(`0x${this.id}`).toString(radix).padStart(length, '0');
+    }
+
+    /**
+     * Converts the high 64 bits of the ID to a string using the specified radix.
+     * @param radix - The base to use for the conversion.
+     * @returns The high 64 bits of the ID as a string in the specified radix.
+     */
+    public highToString(radix: number): string {
+        return BigInt(`0x${this.high}`).toString(radix);
+    }
+
+    /**
+     * Converts the high 64 bits of the ID to a string using the specified radix and pads it with zeros to the specified length.
+     * @param radix - The base to use for the conversion.
+     * @param length - The desired length of the output string.
+     * @returns The high 64 bits of the ID as a padded string in the specified radix.
+     */
+    public highToPaddedString(radix: number, length: number): string {
+        return BigInt(`0x${this.high}`).toString(radix).padStart(length, '0');
+    }
+
+    /**
+     * Converts the low 64 bits of the ID to a string using the specified radix.
+     * @param radix - The base to use for the conversion.
+     * @returns The low 64 bits of the ID as a string in the specified radix.
+     */
+    public lowToString(radix: number): string {
+        return BigInt(`0x${this.low}`).toString(radix);
+    }
+
+    /**
+     * Converts the low 64 bits of the ID to a string using the specified radix and pads it with zeros to the specified length.
+     * @param radix - The base to use for the conversion.
+     * @param length - The desired length of the output string.
+     * @returns The low 64 bits of the ID as a padded string in the specified radix.
+     */
+    public lowToPaddedString(radix: number, length: number): string {
+        return BigInt(`0x${this.low}`).toString(radix).padStart(length, '0');
+    }
+
+    /**
+     * Extracts the Unix timestamp from the 128-bit hex string representation.
+     * @param idHex - The 128-bit ID as a hexadecimal string.
+     * @returns The Unix timestamp as a number.
+     */
+    public static extractTimestamp(idHex: string): number {
+        // Extract the first 8 characters which represent the 32-bit Unix timestamp
+        const timestampHex = idHex.substring(0, 8);
+
+        // Convert the hexadecimal string to a number
+        const timestamp = parseInt(timestampHex, 16);
+
+        return timestamp;
+    }
+
+    /**
+     * Checks if a string representation of an ID in a given radix is within 64 bits.
+     * @param idString - The string representation of the ID.
+     * @param radix - Optional base to use for the conversion (default is 10).
+     * @returns True if the ID is within 64 bits, otherwise false.
+     */
+    static is64Bits(idString: string, radix: number = 10): boolean {
```

Comment:
I'd rename this function -- `is64Bits` to me means it's exactly 64-bits, which isn't what this is testing really. This is `isLessThan64Bits`.